brave/browser-laptop
# Remove smallButton from button.less

The classname has only been used on findbar.

Closes #9232
Addresses #9233

Auditors:

Test Plan:
1. npm run test -- --grep='findBar'
2. Open the browser
3. Display find bar
4. Run manual test plan on https://github.com/brave/browser-laptop/wiki/Manual-Tests#find-on-page

Submitter Checklist:


Test Plan:


Reviewer Checklist:

Tests
